mysql-test/suite/innodb/t/innodb-index.test

```diff
@@ -1202,3 +1202,51 @@ call mtr.add_suppression("Plugin 'InnoDB' init function returned error");
 call mtr.add_suppression("Plugin 'InnoDB' registration as a STORAGE ENGINE failed");
 
 --enable_query_log
+
+--echo #
+--echo #  MDEV-35115 Inconsistent Replace behaviour when multiple
+--echo #               unique index exist
+--echo #
+let $get_handler_status_counts= SELECT * FROM INFORMATION_SCHEMA.SESSION_STATUS WHERE VARIABLE_NAME LIKE 'HANDLER_%' AND VARIABLE_VALUE > 0;
+
+--disable_ps2_protocol
+CREATE TABLE t1 (c1 NUMERIC UNSIGNED NOT NULL,
+		 c2 INT3 UNIQUE,
+		 c3 BIT(2) PRIMARY KEY)ENGINE=InnoDB;
```

Comment:
Do we really have to disable `./mtr --ps-protocol` for the entire test, or would it suffice to do that only around the `eval $get_handler_status_counts`? It should affect `SELECT` only.